Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Input/DeviceManager/Controllers/OculusQuestHand.cs

```diff
@@ -126,6 +126,7 @@ public void CleanupHand()
 
         public override MixedRealityInteractionMapping[] DefaultRightHandedInteractions => DefaultInteractions;
 
+        [System.Obsolete]
         public override void SetupDefaultInteractions(Handedness controllerHandedness)
```

Comment:
Im curious, can this function be removed altogether in MRTK V2.4, or is it only obsolete in 2.5?

Reply:
Its still only obsolete in 2.5, we could change these functions to SetupDefaultInteraction() since we don't use the passed in parameter in any way.

Reply:
Hmm I don't want to start raising errors for users still on 2.4.

If changing to SetupDefaultInteraction() keeps things compatible with 2.4 and 2.5 without raising addtional warnings, that's fine with me.

Reply:
It should be fine since the BaseController already aliases to SetupDefaultInteraction

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/06a06778e38da622b37cc299a93f16e143b7bdeb/Assets/MRTK/Core/Providers/BaseController.cs#L173